services/QuillLMS/client/app/bundles/Evidence/components/activitySlides/postActivitySlide.tsx

```diff
@@ -14,6 +14,45 @@ interface PostActivitySlideProps {
   responses: any
 }
 
+// Helper function that can be exported for testing
+export function getResponsesForConjunction(responses: any, conjunction: string) {
+  const responsesKeys = Object.keys(responses);
+
+  for (const key of responsesKeys) {
```

Comment:
I prefer to use builtin iteration methods rather than `for` loops in general -- since you're not actually using the `key` here except to extract its values, you could turn this whole thing into:

`return Object.values(responses).find(responseArray => ...)`

Your `continue`s will have to become `return false`s, but I think otherwise the logic is the same.

Reply:
Much cleaner